testutils_test.go

```diff
@@ -115,23 +115,22 @@ func benchmarkImmutableAvlTreeWithDB(b *testing.B, db db.DB) {
 	b.StopTimer()
 
 	t := NewMutableTree(db, 100000)
+	value := []byte{}
 	for i := 0; i < 1000000; i++ {
-		t.Set(i2b(int(cmn.RandInt32())), nil)
+		t.Set(i2b(int(cmn.RandInt32())), value)
```

Comment:
`go test -bench=.` was panicking on develop without this change

Reply:
linking https://github.com/tendermint/iavl/issues/76